PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Following that rule breaks the nice convention of having slimmedElectrons -> slimmedElectrons.
What would you suggest here instead?